docs/docs/README.md

```diff
@@ -0,0 +1,581 @@
+# Welcome to WorkFlow Launcher
+
+## Overview
+
+[WorkFlow Launcher (WFL)](https://github.com/broadinstitute/wfl.git) 
+is a workload manager.
+
+It runs as you, with your credentails, from your laptop, and
+communicates with other services as necessary to manage a workload.
+
+It can also be deployed to run as a service in the cloud.
+
+For more on Workflow Launcher's role in the Terra infrastructure see 
+[Workflow Launcher's role in Terra (./docs/docs/terra.md)](./terra.md).
+
+## Set up
+
+Run `boot build` at the top of a `wfl.git` repo to build an
+uberjar. The resulting jar is in `./target/zero-*.jar` relative to
+the `wfl.git` clone.
+
+With some start-up and performance penalty, you can also run 
+Workflow Launcher as a script. See below for details.
+
+## Versioning
+
+Workflow Launcher needs to manage its version and the versions of
+`dsde-pipelines.git` which contribute the WDL files. There may be as
+many `dsde-pipelines.git` versions as there are workflow wdls.
+
+The `wfl` jar includes a manifest with at least that information in
+it, and a version command that returns it.
+
+## Role in Terra
+
+The Data Sciences Platform (DSP) is building a new system (around
```

Comment:
This describes the new role in relation to the old role. Will outside people know what zamboni is? Should they as a prerequisite to understanding what the WFL is? If not, then we shouldn't refer to what zamboni is or does as that will be confusing. Instead we should just say what the WFL does and is.

Reply:
Zamboni and the whole background is explained in the standalone `terra.md`. I think it still makes sense to keep it since a lot of users will still be those who work close with us. What do you think?

Reply:
I can live with it. It's up to what you and the team thinks. Just felt that I should ask the question :)

Reply:
Definitely think that it should be limited to one section though :)